source/_docs/domains.md

```diff
@@ -0,0 +1,182 @@
+
+Pantheon does not allow crawlers on Dev, Test, or Multidev environments. Adding a custom domain to an environment other than Live will not permit crawlers to that environment.
+
+## Custom domains
+Custom domains are domains registered at a third-party provider, for which you control DNS configurations. Connect a custom domain to the Site Dashboard and point DNS at Pantheon to trigger the [automated process of provisioning HTTPS](/docs/https/).
+
+A paid plan is required to connect custom domains to an environment in the Pantheon Site Dashboard, up to the following limits:
+
+<table class="table table-condensed table-bordered">
+  <thead class="thead-inverse">
+    <tr>
+      <th scope="row" class="thead-inverse"></th>
+      <th>Personal</th>
+      <th>Professional</th>
+      <th>Business</th>
+      <th>Elite</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row" class="thead-inverse">Custom Domain Limit (per site)</th>
+      <td>5</td>
+      <td>25</td>
+      <td>100</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <th scope="row" class="thead-inverse">Free and managed HTTPS</th>
+      <td>✓</td>
+      <td>✓</td>
+      <td>✓</td>
+      <td>✓</td>
+    </tr>
+  </tbody>
+</table>
+
+<div class="alert alert-info" role="alert">
+<h4 class="info">Note</h4>
+<p markdown="1">Add all domains you want to resolve to Pantheon within the Site Dashboard for each respective environment, as described in [Launch Essentials](/docs/guides/launch/). Automatic resolution of domains and wildcards are not supported.</p></div>
+
+### Primary domain
+A primary domain refers to a single custom domain used to serve responses, regardless of originating URL of a request (e.g., `https://www.example.com`). Configure the primary domain as part of the going live process described in [Launch Essentials: Redirect to a Primary Domain](/docs/guides/launch/redirects/).
+
+We recommend using the www subdomain prefix for all sites. While it’s good for visitors and DNS to resolve both www and the domain itself, it's best practice to choose one or the other and redirect from www to non-www (or vice versa, your call). This optimizes SEO by avoiding duplicate content and prevents session strangeness, where a user can be logged in to one domain but logged out of other domains at the same time.
+
+## Vanity domains for organizations
+Pantheon Partners, Strategic Partners, Enterprise accounts, Resellers, and OEM Partners have the ability to replace `pantheonsite.io` for each environment on every site they run with a custom vanity domain.
```

Comment:
have the ability > can 
Technically it doesn't "replace" the platform domain. I think the language in /docs/vanity-domains might work